game/scripts/npc/abilities/talents/visage_talent4.txt

```diff
@@ -0,0 +1,26 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Bonus Chilling Touch Damage
+  //=================================================================================================================
+  "special_bonus_unique_visage_4"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "ID"					""														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
+    "AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
+    "AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+
+    // Special
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilitySpecial"
+    {
+      "01"
+      {
+        "var_type"					"FIELD_INTEGER"
+        "value"				"40" //OAA
```

Comment:
I wouldn't play with this talent. Same reasons as Gyro's talent. This spells works weird so even a slight buff is really good. Set to 30 or revert.